### PR TITLE
[WIP] Steps to configure

### PR DESCRIPTION
**Pull request by joecohens**
Not sure if it would be better to store checks on settings like `has_setup_components`, `has_setup_incidents`, `has_setup_customize`. So we can set a notification bar at the top to help with the steps your missing.

screen shot 2015-01-13 at 12 55 18 am

Closes https://github.com/cachethq/Cachet/issues/99

### CODE REVIEW

**Comment by jbrooksuk** [comment]
@joecohens you'll also need to copy the `en` translation to the other three to keep them in sync for now. When we're closer to finishing I'll push everything to be translated.

**Comment by jbrooksuk** [comment]
With that said @joecohens all of the text which is within the views need moving into the `./app/lang/en/` files :)

**Comment by joecohens** [comment]
@jbrooksuk Yeah I haven't  done the translations yet, will do them now. What do you think about the way I'm checking if you have done steps? Or do you like better the proposal I've made on the comment?

File: src/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -24,6 +26,101 @@ public function showDashboard()
...
+     * Shows missing configurations view.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function showConfiguration()
+    {
+        $hasComponents = (Component::withTrashed()->count() > 0);
+        $hasIncidents = (Incident::withTrashed()->count() > 0);
+        $hasCustomized = (Setting::get('style_background_color') || Setting::get('style_text_color'));
+
```
Comment (joecohens) [inline src/Http/Controllers/DashboardController.php]: @cachethq/owners Not sure about this checks what do you think? Maybe we could have a Setting for each check like `has_customized => true`?

Comment (jbrooksuk) [inline src/Http/Controllers/DashboardController.php]: That's not a bad idea.

Comment (jbrooksuk) [inline src/Http/Controllers/DashboardController.php]: Also made better by your new setting caching.

Comment (jbrooksuk) [inline src/Http/Controllers/DashboardController.php]: Also, the `$hasCustomized` could include a custom stylesheet.

**Comment by jbrooksuk** [comment]
@joecohens any update on this? It's so close :)

**Comment by joecohens** [comment]
See https://github.com/cachethq/Cachet/pull/403

**Pull request closed without merging**